utilities/dataflow_hcapi_to_gcs_pubsub/dataflow_hcapi_to_gcs_pubsub.py

```diff
@@ -0,0 +1,380 @@
+        data = json.dumps(newElement).encode('utf-8')
+        pubsubMessage = io.gcp.pubsub.PubsubMessage(data=data,attributes=attributes)
+        logging.info(str(pubsubMessage))
+        yield pubsubMessage
+    
+
+def run(argsDict,beam_args,argv=None):
+    beam_pipeline_options = PipelineOptions(beam_args,streaming=True,save_main_session=True)
+    read_pubsub_subscription = str(argsDict['read_pubsub_subscription']).split(",")
+    subscription1 = io.gcp.pubsub.PubSubSourceDescriptor(str(read_pubsub_subscription[0]))
+    subscription2 = io.gcp.pubsub.PubSubSourceDescriptor(str(read_pubsub_subscription[1]))
+    write_pubsub_topic = str(argsDict['write_pubsub_topic'])
+    gcs_archive_bucket = str(argsDict['gcs_archive_bucket'])
+    gcs_error_bucket = str(argsDict['gcs_error_bucket'])
+    final_fhir_store= str(argsDict['hcapi_get_fhir_store'])
+    
+    with beam.Pipeline(options=beam_pipeline_options) as pipeline:
+        consumeMessages = (
+            pipeline
+            | "readMessageFromPubSub" >> io.MultipleReadFromPubSub([subscription1,subscription2],with_attributes=True)
+            | 'transformRawFHIRMessage' >> beam.ParDo(consumeTransformMessages()).with_outputs()
+            )
+        WriteSuccessRecordsToGCS = (
+            consumeMessages.successRecord
+            | "archiveRawFHIRMessageToGCS" >> beam.ParDo(writeToGCS(),topic=final_fhir_store,bucket=gcs_archive_bucket,messageType="raw_fhir_resources")
+        )
+        writeFailRawRecords = (
+            consumeMessages.failRecord
+            | "writeFailRawRecordsToGCS" >> beam.ParDo(writeToGCS(),topic="fail_transform_raw_messages",bucket=gcs_error_bucket,messageType="error")
+            )
+        processSuccessRecords = (
+            consumeMessages.successRecord
+            | "buildPubSubMesage" >> beam.ParDo(buildPubSubMesage())
+            | "writeToPubSub" >> beam.io.gcp.pubsub.WriteToPubSub(topic=write_pubsub_topic,with_attributes=True)
+            )
+
+        
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s :: %(levelname)-8s :: [%(filename)s:%(lineno)d] :: %(message)s')
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--read_pubsub_subscription', required=True, help='Pub/Sub Subscription to consume messages')
+    parser.add_argument('--write_pubsub_topic', required=True, help='Pub/Sub Subscription to consume messages')
+    parser.add_argument('--hcapi_project_id', required=True, help='HCAPI project ID')
+    parser.add_argument('--hcapi_dataset', required=True, help='HCAPI dataset')
+    parser.add_argument('--hcapi_version', required=True, help='HCAPI Version')
+    parser.add_argument('--hcapi_location', required=True, help='HCAPI location')
+    parser.add_argument('--hcapi_get_fhir_store', required=False, help='HCAPI FHIR Store to GET FHIR resources')
+    parser.add_argument('--hcapi_post_fhir_store', required=True, help='HCAPI FHIR Store to POST FHIR resources')
```

Comment:
update help for write_pub_topic
for hcapi variables accept full qualified FHIR store name rather building URL in code.

Reply:
Updated the help